planner/core/casetest/testdata/expression_rewriter_suite_out.json

```diff
@@ -21,7 +21,7 @@
       {
         "SQL": "select * from t1 where (t1.a, t1.b) not in (select a, b from t2)",
         "Plan": [
-          "HashJoin 3.20 root  CARTESIAN anti semi join, other cond:eq(test.t1.a, test.t2.a), eq(test.t1.b, test.t2.b)",
+          "HashJoin 3.20 root  Null-aware anti semi join, equal:[eq(test.t1.b, test.t2.b) eq(test.t1.a, test.t2.a)]",
```

Comment:
Why the key order is changed

Reply:
Seems the order has all changed. @AilinKid any idea?

Reply:
I find the root cause and fix it.

Reply:
I disorder the keys, just a mistake, but won't affect the result